Student/Widgets/Resources/Assets.xcassets/AccentColor.colorset/Contents.json

```diff
@@ -0,0 +1,11 @@
+{
+  "colors" : [
+    {
+      "idiom" : "universal"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
+}
```

Comment:
I don't see any actual colors defined here, plus the name would get confusing with SwiftUI's `Color.accentColor` which basically inherits & resolves like `tintColor` does in UIKit.

Reply:
Nice catch. This was a copy-paste error.

Reply:
Seems like these colors are required by the Add Widget iOS screen. These influence how the "+ Add Widget" button and the back of the widget (in case there are configurable parameters you can flip the widget to set those) are colored. 
Removing gives two Xcode warnings so I'll just leave them as-is with the default values.
<img width="335" alt="Screenshot 2020-11-12 at 10 52 18" src="https://user-images.githubusercontent.com/72396990/98928985-cadec680-24da-11eb-91a2-b3d32b49f329.png">